src/FeigenbaumCore.jl

```diff
@@ -37,9 +38,9 @@ end
 Return true if permutation perm has subgraph of period n
 
 # Example
-```julia
- julia> hasperiodn([4, 5, 7, 6, 3, 2, 1], 5)
- false
+```jldoctest
+julia> using FeigenbaumUtil; hasperiodn([4, 5, 7, 6, 3, 2, 1], 5)
```

Comment:
Might improve test perf, but definitely reduce noise if you use setup code https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Setup-Code-1) instead of `using` each time